### DISCUSSION TITLE
Is there a desktop client for cockpit?

### DISCUSSION THREAD

**Discussion by jaitra1264** [cockpit-project/cockpit#16773] [Q&A] [answered]

**Comment by marusak** [comment, marked as answer]
Just a few weeks ago we released flatpak cockpit client, which lets you to log into any machine (local or remote) - https://cockpit-project.org/blog/cockpit-259.html
There is also cockpit-desktop, which connects you to already running local session - https://cockpit-project.org/guide/latest/cockpit-desktop.1.html
